src/components/ActionCards/index.js

```diff
@@ -9,8 +9,8 @@ const renderIcon = Icon => (
 
 const ActionCards = ({ actionCardItems }) => (
   <div className="card-wrapper">
-    {actionCardItems.map(item => (
-      <div className="card-container">
+    {actionCardItems.map((item, index) => (
```

Comment:
Isn't using the index as a key meant to be bad practise when mapping elements in react ? Since our data in a constant file and not going to change, maybe we could use the `title` attribute ?